portable/NetworkInterface/STM32Hxx/NetworkInterface.c

```diff
@@ -349,71 +349,74 @@ BaseType_t xNetworkInterfaceOutput( NetworkBufferDescriptor_t * const pxDescript
     TickType_t xBlockTimeTicks = pdMS_TO_TICKS( 100U );
     uint8_t * pucTXBuffer;
 
-    #if ( ipconfigZERO_COPY_TX_DRIVER != 0 )
-        /* Zero-copy method, pass the buffer. */
-        pucTXBuffer = pxDescriptor->pucEthernetBuffer;
-
-        /* As the buffer is passed to the driver, it must exist.
-         * The library takes care of this. */
-        configASSERT( xReleaseAfterSend != pdFALSE );
-    #else
-        pucTXBuffer = Tx_Buff[ xEthHandle.TxDescList.CurTxDesc ];
-        /* The copy method, left here for educational purposes. */
-        configASSERT( pxDescriptor->xDataLength <= sizeof( Tx_Buff[ 0 ] ) );
-    #endif
-
-    ETH_BufferTypeDef xTransmitBuffer =
+    if( xGetPhyLinkStatus() == = pdPASS )
```

Comment:
I am seeing an extra 'equal-to' (`=`) sign here.

Reply:
Yes, my compiler saw that too! I only forgot to submit that change.
Done so now, along with 3 minor changes.